### PR TITLE
fix: WT-2085 - patched onramp

### PR DESCRIPTION
**Pull request by andrearampin**
# Summary

We were showing the loading screen too early in the process and therefore the 3DS check screen was getting obfuscated by a Checkout (`IN_PROGRESS`) screen preventing the user to complete the onramp.

https://github.com/immutable/ts-immutable-sdk/assets/4301501/0ba46335-bb7d-449c-bf28-f0eb74c8fd75

# Impact of Change

Enable onramp with user verifications.

## Fixed

Transak event listening

# Other things worth calling out

# Before submitting the PR, follow these steps:

- [ ] Prefix your PR title with `feat: `, `fix: `, `chore: `, `docs:`, or `refactor:`.
- [ ] Prefix your PR title with `breaking-change:` if you have introduced breaking changes to this SDK. Breaking changes are modification that necessitates immediate adjustments by this SDK's users to their applications, clients, or integrations to avert disruptions to existing features or functionalities. Ensure you write clear summary explain the change.

### CODE REVIEW

File: packages/checkout/widgets-lib/src/widgets/on-ramp/views/OnRampMain.tsx [hunk 1]
```diff
@@ -121,7 +121,11 @@ export function OnRampMain({
       return;
     }
 
-    if (event.event_id === TransakEvents.TRANSAK_ORDER_CREATED) {
```
Comment (andrearampin) [inline packages/checkout/widgets-lib/src/widgets/on-ramp/views/OnRampMain.tsx:124]: `TransakEvents.TRANSAK_ORDER_CREATED` still used in Segment event tracking

**Review by dreamoftrees — APPROVED** [review]
Nice find @andrearampin .. does that mean that we will always have a 6sec delay for the progress screen?

**Review by jhesgodi — APPROVED** [review]
Thanks for fixing this!

**Comment by andrearampin** [comment]
> Nice find @andrearampin .. does that mean that we will always have a 6sec delay for the progress screen?

Yes, the UX looked broken without that delay and I figured this is matching what Cass proposed.

**Pull request merged**